pointers/pointers.c

```diff
@@ -21,7 +29,13 @@ void swap(int* a, int* b)
 */
 char *find_char(char *str, int c)
 {
-
+    while (*str != c || *str != '\0') {
+        str++;
+    }
+    if (*str == 0) {
```

Comment:
I think you're checking for the null terminator here. Make sure you check for `'\0'`, not `0`!